openml/runs/functions.py

```diff
@@ -85,6 +81,19 @@ def _run_task_get_arffcontent(model, task, class_labels):
             testY = Y[test_indices]
 
             model.fit(trainX, trainY)
+            if isinstance(model, BaseSearchCV):
```

Comment:
I think it would be good to have this in it's own function, both for readability and for testability.